frontend/src/components/elements/BokehChart/mock.ts

```diff
@@ -0,0 +1,232 @@
+export default {
```

Comment:
Add header.

Also add a comment explaining what this is. (I'm imagining this is a serialized Bokeh chart?)

Reply:
Done